### ISSUE TITLE
Raise fixme warnings for lowercase 'todo' comments

### ISSUE THREAD

**Issue by Stevoisiak** [PyCQA/pylint#2029]
Pylint will not raise `fixme` warnings if a `todo` comment is written with lowercase letters.

### Steps to reproduce
1. Open any Python script
2. Add a lowercase todo comment.

        # todo: add more features

3. Run Pylint

### Current behavior

No warnings are raised.

### Expected behavior

A `fixme` error should be raised. (Same behavior as `TODO` comments)

    W:  1, 0: todo: add more features (fixme)

### pylint --version output

```
No config file found, using default configuration
pylint 1.8.3,
astroid 1.6.2
Python 3.6.4 (v3.6.4:d48eceb, Dec 19 2017, 06:04:45) [MSC v.1900 32 bit (Intel)]
```
